transmart-data/ddl/postgres/deapp/de_subject_sample_mapping.sql

```diff
@@ -47,3 +47,15 @@ CREATE INDEX de_subject_smpl_mpng_idx2 ON de_subject_sample_mapping USING btree
 --
 CREATE INDEX idx_de_subj_smpl_trial_ccode ON de_subject_sample_mapping USING btree (trial_name, concept_code);
 
+--
+-- add documentation
+--
+COMMENT ON TABLE deapp.de_subject_sample_mapping IS 'Table links patients to Highdim samples';
+
+COMMENT ON COLUMN de_subject_sample_mapping.patient_id IS 'The patient id linking the patient_dimension';
+COMMENT ON COLUMN de_subject_sample_mapping.subject_id IS 'links to subject_id in de_subject_microarray_data';
+COMMENT ON COLUMN de_subject_sample_mapping.concept_code IS 'The highdim concept code. E.g. CTHD:HD:EXPLUNG';
+COMMENT ON COLUMN de_subject_sample_mapping.assay_id IS 'Id that links to assay_id in highdim tables like de_subject_microarray_data and de_rnaseq_transcript_data';
```

Comment:
Suggestion:
Used as primary key of this table (although it is not an actual primary key and there is not even a proper index for this column). This key is references by high dimensional data tables, like de_subject_microarray_data and de_rnaseq_transcript_data.